app/models/bucket.rb

```diff
@@ -0,0 +1,22 @@
+class Bucket
+  attr_reader :contents
+
+  def initialize(contents)
+    @contents = contents || {}
+  end
+
+  def add_job(job_id)
+    @contents[job_id.to_s] ||= ""
+  end
+
+  def total_jobs
+    contents.length
+  end
+
+  def all_items #edit this method and write test
```

Comment:
@roscalabrin Make sure this comment doesn't make it into master. Either complete the task, or make a note in a Waffle card explaining what this means. I actually am not sure what `edit this method` means haha.